src/main.rs

```diff
@@ -95,8 +96,16 @@ fn main() {
    );
 
    let json = serde_json::to_string(&reading).unwrap();
+   let url = "http://localhost:4000/api/readings/add";
```

Comment:
Do you ever mean for this url to change? If not, I recommend that you use `const`.

Reply:
Eventually the URL will come from user input so will be dynamic. In my head that's what I was thinking when I wrote this. It makes a ton of sense to use `const` until that time. Thank you! Update in 87c0320e52e8b9d3fedbc1ca8ec76d4ebc2a26ee